Library/Homebrew/test/formula_spec.rb

```diff
@@ -699,6 +699,38 @@
     end
   end
 
+  specify "#service" do
+    f = formula do
+      url "https://brew.sh/test-1.0.tbz"
+      service do
+        run ["/usr/local/bin/beanstalkd"]
+        run_type "immediate"
+        error_log_path "/var/log/beanstalkd.error.log"
+        log_path "/var/log/beanstalkd.log"
+        working_dir "/var"
+        keep_alive true
+      end
+    end
+
+    expect(f.service.run).to eq(["/usr/local/bin/beanstalkd"])
+    expect(f.service.error_log_path).to eq("/var/log/beanstalkd.error.log")
+    expect(f.service.log_path).to eq("/var/log/beanstalkd.log")
+    expect(f.service.working_dir).to eq("/var")
+    expect(f.service.keep_alive).to eq(true)
+    expect(f.service.run_type).to eq("immediate")
+  end
+
+  specify "service uses simple run" do
+    f = formula do
+      url "https://brew.sh/test-1.0.tbz"
+      service do
+        run "/usr/local/bin/beanstalkd"
+      end
```

Comment:
```suggestion
      service "/usr/local/bin/beanstalkd"
```

I wonder if that'd be any good as short-hand?

Reply:
I haven't really seen any plists that define just a run command. But if you think it's useful it could be added.

Reply:
Nah, I defer to you there, let's not add it until we need it.